core/modules/replica/ControllerApp.cc

```diff
@@ -95,6 +98,42 @@ void printRequest(Request::Ptr const& request,
 }
 
 
+void printRequest(IndexRequest::Ptr const& request, string const& fileName) {
+    auto&& responseData = request->responseData();
+    cout << request->context() << "\n"
+         << "responseData.error: " << responseData.error << "\n"
+         << "performance:  " << request->performance() << endl;
+
+    if (fileName.empty()) cout << responseData.data;
```

Comment:
is it ok to mix single-line if with multi-line, braced, else? (I wouldn't think so...)

Reply:
👍